README.md

```diff
@@ -129,3 +129,23 @@ mode using `webpack -w`.
   Defaults to `firefox-desktop`.
 
   See the documentation for the `--target` option of [`web-ext run`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-run).
+
+### Android-specific options
+
+These options only apply when `target` is set to `firefox-android`:
```

Comment:
```suggestion
These options only apply when `target` includes `firefox-android`:
```

I think `target` can take an array of target types.

Reply:
Yep you are totally right